head.html

```diff
@@ -10,4 +10,5 @@
 </script>
 <script src="/scripts/aem.js" type="module"></script>
 <script src="/scripts/scripts.js" type="module"></script>
+<!-- <link rel="stylesheet" href="https://cdn.udex.services.sap.com/dds/design-tokens/v0.36.0/css/sap_glow.css"/> -->
```

Comment:
Why would this comment be left in head.html?

Reply:
Good catch, removed it